src/components/Map/Map.js

```diff
@@ -40,23 +40,23 @@ function Map({ hideLegend = false, setMobileMenuOpen, setMapOption }) {
         <Legend>
           <LegendItem
             key={'legend-3'}
-            title={'Cases are increasing'}
+            title={'No criteria met for reopening'}
             color={COLOR_MAP.RED.BASE}
             description={
               'Hospitals at elevated risk to be overloaded in the next 3 weeks. Act now to flatten the curve.'
             }
           />
           <LegendItem
             key={'legend-2'}
-            title={'Cases are stable'}
+            title={'Some criteria met for reopening'}
             color={COLOR_MAP.ORANGE.BASE}
             description={
               'Hospitals at moderate risk to be overloaded in the next 3-6 weeks.'
             }
           />
           <LegendItem
             key={'legend-1'}
-            title={'Cases are decreasing'}
+            title={'All criteria met for reopening'}
```

Comment:
FYI- I'm nervous about using language that's too strong here.  But I assume we'll review / fine-tune.